package.json

```diff
@@ -71,7 +73,7 @@
     "webpack-merge": "^4.2.2"
   },
   "dependencies": {
-    "@centreon/ui": "github:centreon/centreon-ui#MON-5014-filter-style",
+    "@centreon/ui": "github:centreon/centreon-ui#adbb81c3b4dc6bf67842b890c2ac4e175562decd",
```

Comment:
Change are not retro-compatible. I'll handle the changes from the latest version in the next Unified view PR

Reply:
will have to think about it though :)